web/lib/ozoneframework/php/core/RunData.php

```diff
@@ -597,9 +596,8 @@ public function handleSessionEnd() {
 					$session->setSessionChanged(false);
 				}
 
-				$mc = OZONE::$memcache;
 				$key = 'session..'.$session->getSessionId();
-				$mc->set($key, $session, 0, 600);
+				Cache::put($key, $session,600);
```

Comment:
```suggestion
				Cache::put($key, $session, 600);
```

Reply:
Sorry, these are hard to tell sometimes in the IDE as it inserts the arg name and it looks like there's a space there.